### DISCUSSION TITLE
User with create_export perms but not view_other_timesheet exports all users

### DISCUSSION THREAD

**Discussion by TyroneRussellESDI** [kimai/kimai#4671] [General] [answered]
Hi! I'm trying out Kimai 2. Got it installed and my Kimai 1 data imported. 

I'm trying to setup so a user can export their timesheet entries to a spreadsheet. The issue I'm having is that if you give the user create_export permission, their export contains records from all users. (No other permission changes from default after the install and import)

If you give that user the view_other_timesheet permission, they then gain a dropdown on the export window that allows them to filter down to only their records (or the records of another user). My expectation would be that if a user doesn't have "view_other_timesheet" that it would be ok for them to not have a user select because they should only be able to export their records, but that is not what is happening. The user is instead forced to export all user's records, even the records from users they don't have permission to view, and has no way to filter to only their records.

My question: Is this a bug, do I have something installed wrong, is it possible the kimai1 import left me with poor permissions?

Edit: 2.12.0 installed on a shared hosting server

**Comment by kevinpapst** [comment, marked as answer]
Read: https://www.kimai.org/documentation/export.html

This is documented behavior. There is an export button in the users own timesheet view for your purpose

**Reply by TyroneRussellESDI** [reply]
Thank you so much for the help! I appreciate your reading through a confusion that I (a new user) had and helping.
